lib/src/integrationTest/java/io/github/taz03/jia/friendships/FriendshipsActionTest.java

```diff
@@ -0,0 +1,27 @@
+package io.github.taz03.jia.friendships;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import io.github.taz03.jia.TestProxy;
+import io.github.taz03.jia.requests.friendships.FriendshipsActionRequest;
+import io.github.taz03.jia.responses.friendships.FriendshipsActionResponse;
+import org.junit.jupiter.api.Test;
+
+public class FriendshipsActionTest {
+    private final long pk = 25025320;
```

Comment:
static, name it `INSTAGRAM_PK` and explain in a comment maybe also give the link to profile

Reply:
i see you already change it at main commits. I do git pull. Is it right?

Reply:
This pr is not conflicting  so if u want  u can continue, but if u want to rebase first checkout `main` then do fetch and pull after that checkout this branch again and do a merge or rebase. But that is not required